client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table/index.tsx

```diff
@@ -39,77 +45,127 @@ const SiteTable = ( { isLoading, columns, items }: Props, ref: Ref< HTMLTableEle
 		showLicenseInfo( license );
 	};
 
-	return (
-		<table ref={ ref } className="site-table__table">
-			<thead>
-				<tr>
-					{ isBulkManagementActive ? (
-						// Take full-width of the table
-						<th colSpan={ 100 }>
-							<SiteBulkSelect isLargeScreen sites={ items } isLoading={ isLoading } />
-						</th>
-					) : (
-						<>
-							{ columns.map( ( column, index ) => (
-								<th key={ column.key }>
-									<SiteSort isLargeScreen isSortable={ column.isSortable } columnKey={ column.key }>
-										{ index === 0 && (
-											<Icon className="site-table__favorite-icon" size={ 24 } icon={ starFilled } />
-										) }
-										<span className={ classNames( index === 0 && 'site-table-site-title' ) }>
-											{ column.title }
+	const hasAddNewSiteTourPreference = useSelector( ( state ) =>
+		getPreference( state, 'jetpack-cloud-site-dashboard-add-new-site-tour' )
+	);
+
+	const latestJetpackConnectedSite = localStorage.getItem( 'latestJetpackConnectedSite' )
+		? localStorage.getItem( 'latestJetpackConnectedSite' )
+		: '';
 
-											{ column.showInfo && (
+	const shouldRenderAddSiteTourStep2 =
+		hasAddNewSiteTourPreference && latestJetpackConnectedSite !== '';
+
+	const tourHTMLTarget =
+		items.length < 20
+			? 'tr.is-latest-jetpack-connected-site td:first-of-type'
+			: '.site-table__table th:first-of-type';
```

Comment:
I still disagree with targeting anything other than the site in question in its current state, as otherwise the messaging is just wrong:

`🎉 Your new site is here`
`Check out your new site here. That was straightforward, right?`

Reply:
We will leave as it is and bring your concern to design in their final review. Does that work for you?